### ISSUE TITLE
2025-01-18, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#55]
# Today's Arxiv Papers

(1) [Distilling Multi-modal Large Language Models for Autonomous Driving](http://arxiv.org/abs/2501.09757v1), 2025-01-16

(2) [Lost in Translation, Found in Context: Sign Language Translation with Contextual Cues](http://arxiv.org/abs/2501.09754v1), 2025-01-16

(3) [OmniThink: Expanding Knowledge Boundaries in Machine Writing through Thinking](http://arxiv.org/abs/2501.09751v1), 2025-01-16

(4) [Enhancing Lexicon-Based Text Embeddings with Large Language Models](http://arxiv.org/abs/2501.09749v1), 2025-01-16

(5) [Suggesting Code Edits in Interactive Machine Learning Notebooks Using Large Language Models](http://arxiv.org/abs/2501.09745v1), 2025-01-16

(6) [Inference-Time Scaling for Diffusion Models beyond Scaling Denoising Steps](http://arxiv.org/abs/2501.09732v1), 2025-01-16

(7) [CyberMentor: AI Powered Learning Tool Platform to Address Diverse Student Needs in Cybersecurity Education](http://arxiv.org/abs/2501.09709v1), 2025-01-16

(8) [Domain Adaptation of Foundation LLMs for e-Commerce](http://arxiv.org/abs/2501.09706v1), 2025-01-16

(9) [Simulated Interactive Debugging](http://arxiv.org/abs/2501.09694v1), 2025-01-16

(10) [Towards Large Reasoning Models: A Survey of Reinforced Reasoning with Large Language Models](http://arxiv.org/abs/2501.09686v1), 2025-01-16

(11) [Robin: a Suite of Multi-Scale Vision-Language Models and the CHIRP Evaluation Benchmark](http://arxiv.org/abs/2501.09672v1), 2025-01-16

(12) [A Survey of Research in Large Language Models for Electronic Design Automation](http://arxiv.org/abs/2501.09655v1), 2025-01-16

(13) [The Heap: A Contamination-Free Multilingual Code Dataset for Evaluating Large Language Models](http://arxiv.org/abs/2501.09653v1), 2025-01-16

(14) [CarMem: Enhancing Long-Term Memory in LLM Voice Assistants through Category-Bounding](http://arxiv.org/abs/2501.09645v1), 2025-01-16

(15) [LLM-Based Routing in Mixture of Experts: A Novel Framework for Trading](http://arxiv.org/abs/2501.09636v1), 2025-01-16

(16) [Empowering Large Language Models in Wireless Communication: A Novel Dataset and Fine-Tuning Framework](http://arxiv.org/abs/2501.09631v1), 2025-01-16

(17) [Beyond Reward Hacking: Causal Rewards for Large Language Model Alignment](http://arxiv.org/abs/2501.09620v1), 2025-01-16

(18) [From Scarcity to Capability: Empowering Fake News Detection in Low-Resource Languages with LLMs](http://arxiv.org/abs/2501.09604v1), 2025-01-16

(19) [AI in Support of Diversity and Inclusion](http://arxiv.org/abs/2501.09534v1), 2025-01-16

(20) [AdaFV: Accelerating VLMs with Self-Adaptive Cross-Modality Attention Mixture](http://arxiv.org/abs/2501.09532v1), 2025-01-16

(21) [Confidence Estimation for Error Detection in Text-to-SQL Systems](http://arxiv.org/abs/2501.09527v1), 2025-01-16

(22) [Augmenting a Large Language Model with a Combination of Text and Visual Data for Conversational Visualization of Global Geospatial Data](http://arxiv.org/abs/2501.09521v1), 2025-01-16

(23) [Omni-Emotion: Extending Video MLLM with Detailed Face and Audio Modeling for Multimodal Emotion Analysis](http://arxiv.org/abs/2501.09502v1), 2025-01-16

(24) [Evaluating Conversational Recommender Systems with Large Language Models: A User-Centric Evaluation Framework](http://arxiv.org/abs/2501.09493v1), 2025-01-16

(25) [Exploring the Inquiry-Diagnosis Relationship with Advanced Patient Simulators](http://arxiv.org/abs/2501.09484v1), 2025-01-16

(26) [Guided Debugging of Auto-Translated Code Using Differential Testing](http://arxiv.org/abs/2501.09475v1), 2025-01-16

(27) ["A Great Start, But...": Evaluating LLM-Generated Mind Maps for Information Mapping in Video-Based Design](http://arxiv.org/abs/2501.09457v1), 2025-01-16

(28) [Solving the unsolvable: Translating case law in Hong Kong](http://arxiv.org/abs/2501.09444v1), 2025-01-16

(29) [A Survey on Responsible LLMs: Inherent Risk, Malicious Use, and Mitigation Strategy](http://arxiv.org/abs/2501.09431v1), 2025-01-16

(30) [AutoCBT: An Autonomous Multi-agent Framework for Cognitive Behavioral Therapy in Psychological Counseling](http://arxiv.org/abs/2501.09426v1), 2025-01-16

(31) [FASP: Fast and Accurate Structured Pruning of Large Language Models](http://arxiv.org/abs/2501.09412v1), 2025-01-16

(32) [MoE$^2$: Optimizing Collaborative Inference for Edge Large Language Models](http://arxiv.org/abs/2501.09410v1), 2025-01-16

(33) [Evaluating LLM Abilities to Understand Tabular Electronic Health Records: A Comprehensive Study of Patient Data Extraction and Retrieval](http://arxiv.org/abs/2501.09384v1), 2025-01-16

(34) [Adaptive Contextual Caching for Mobile Edge Large Language Model Service](http://arxiv.org/abs/2501.09383v1), 2025-01-16

(35) [Aligning Instruction Tuning with Pre-training](http://arxiv.org/abs/2501.09368v1), 2025-01-16

(36) [PICE: A Semantic-Driven Progressive Inference System for LLM Serving in Cloud-Edge Networks](http://arxiv.org/abs/2501.09367v1), 2025-01-16

(37) [YETI (YET to Intervene) Proactive Interventions by Multimodal AI Agents in Augmented Reality Tasks](http://arxiv.org/abs/2501.09355v1), 2025-01-16

(38) [Rational Tuning of LLM Cascades via Probabilistic Modeling](http://arxiv.org/abs/2501.09345v1), 2025-01-16

(39) [SOP-Agent: Empower General Purpose AI Agent with Domain-Specific SOPs](http://arxiv.org/abs/2501.09316v1), 2025-01-16

(40) [A Study of In-Context-Learning-Based Text-to-SQL Errors](http://arxiv.org/abs/2501.09310v1), 2025-01-16

(41) [RoboReflect: Robotic Reflective Reasoning for Grasping Ambiguous-Condition Objects](http://arxiv.org/abs/2501.09307v1), 2025-01-16

(42) [To Retrieve or Not to Retrieve? Uncertainty Detection for Dynamic Retrieval Augmented Generation](http://arxiv.org/abs/2501.09292v1), 2025-01-16

(43) [LAVCap: LLM-based Audio-Visual Captioning using Optimal Transport](http://arxiv.org/abs/2501.09291v1), 2025-01-16

(44) [Large Language Model is Secretly a Protein Sequence Optimizer](http://arxiv.org/abs/2501.09274v1), 2025-01-16

(45) [Perspective Transition of Large Language Models for Solving Subjective Tasks](http://arxiv.org/abs/2501.09265v1), 2025-01-16

(46) [Delayed Fusion: Integrating Large Language Models into First-Pass Decoding in End-to-end Speech Recognition](http://arxiv.org/abs/2501.09258v1), 2025-01-16

(47) [Clone-Robust AI Alignment](http://arxiv.org/abs/2501.09254v1), 2025-01-16

(48) [Split Fine-Tuning for Large Language Models in Wireless Networks](http://arxiv.org/abs/2501.09237v1), 2025-01-16

(49) [Foundations of Large Language Models](http://arxiv.org/abs/2501.09223v1), 2025-01-16

(50) [A Simple Graph Contrastive Learning Framework for Short Text Classification](http://arxiv.org/abs/2501.09219v1), 2025-01-16

(51) [Interpretable Droplet Digital PCR Assay for Trustworthy Molecular Diagnostics](http://arxiv.org/abs/2501.09218v1), 2025-01-16

(52) [Boosting Short Text Classification with Multi-Source Information Exploration and Dual-Level Contrastive Learning](http://arxiv.org/abs/2501.09214v1), 2025-01-16

(53) [FineMedLM-o1: Enhancing the Medical Reasoning Ability of LLM from Supervised Fine-Tuning to Test-Time Training](http://arxiv.org/abs/2501.09213v1), 2025-01-16

(54) [Personalized Parsons Puzzles as Scaffolding Enhance Practice Engagement Over Just Showing LLM-Powered Solutions](http://arxiv.org/abs/2501.09210v1), 2025-01-16
